kythe/extractors/bazel/extract.sh

```diff
@@ -86,5 +86,14 @@ fi
 mkdir -p "$KYTHE_OUTPUT_DIRECTORY"
 find bazel-out/*/extra_actions/external/kythe_release -name '*.kzip' | \
   xargs -r /kythe/tools/kzip merge --append --encoding "$KYTHE_KZIP_ENCODING" --output "$KYTHE_OUTPUT_DIRECTORY/compilations.kzip"
+
+# Record the timestamp of the git commit in a metadata kzip.
+METADATA_KZIP="buildmetadata.kzip"
```

Comment:
Curious what the value of having this be an env var is?

Reply:
Not much, I removed it. Before I re-arranged this a bit, there was more distance between the creation of the metadata kzip and the merge step, so it made more sense then.